src/macros.rs

```diff
@@ -354,39 +354,48 @@ macro_rules! BITFIELD {
         )+}
     }
 }
+
 #[macro_export]
+#[allow(non_snake_case)]
 macro_rules! ENUM {
```

Comment:
macro names should be in lowercase too. maybe it's more fitting to name this macro `win_enum` or `c_enum`